ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,37 @@
+<!-- Thank you for contributing to Crust firmware!
+
+Issues that do not conform to one of the templates below are subject to
+immediate closure.
+
+FOR BUG REPORTS:
+Please describe, using as much detail as possible, the issue encountered, and
+the steps necessary to reproduce the issue. It is also crucial that all
+relevant output and toolchain versions are included.
+
+If you are using the Crust build system from the meta repo, please include the
+configuration section of the Makefile.
+
+FOR FEATURE REQUESTS:
+Please describe, using as much detail as possible, the feature you would like
+to see introduced and how you will benefit from its addition.
+
+Additionally, please include if this is a feature you can contribute to adding.
```

Comment:
s/include/indicate
feature +where you can contribute to ~~adding~~ the implementation